src/datasource.ts

```diff
@@ -85,30 +85,36 @@ export class DataSource extends DataSourceWithBackend<Query, HistorianDataSource
               assetMeasurementQuery.Options,
               request.scopedVars
             )
+            assetMeasurementQuery.Options.ValueFilters = assetMeasurementQuery.Options.ValueFilters?.filter(
+              (e) => e.Value !== 'enter a value'
+            )
```

Comment:
this seems a bit hacky.... How come a place holder value is filter out here?

Reply:
was already done like this for the other tag filters. Placeholder values indicate that filter is not filled out completely

Reply:
but yea, I guess the string value 'enter a value' specifically will not work :shrug:

Reply:
we should rework this asap

Reply:
I'll put it in another story